### DISCUSSION TITLE
How to get the metadata of the video?

### DISCUSSION THREAD

**Discussion by Anudnya93** [MetaLabs-inc/react-native-vimeo-iframe#42] [General]
I have tried using loadedmetadata inside the handlers props. But there aren't any logs getting printed. Kindly help.

  const videoCallbacks = {
    timeupdate: data => console.log('timeupdate: ', data),
    play: data => console.log('play: ', data),
    pause: data => console.log('pause: ', data),
    fullscreenchange: data => console.log('fullscreenchange: ', data),
    ended: data => console.log('ended: ', data),
    controlschange: data => console.log('controlschange: ', data),
    loadedmetadata: data => console.log('loadedmetadata', data),
    loadeddata: data => console.log('loadeddata', data)
  }

return (
      <Vimeo
        videoId={'741193833'}
        params={'api=1&autoplay=0'}
          ={videoCallbacks}
        containerStyle={{
          height: 150,
          width: '100%'
        }}
        otherProps={{ allowsInlineMediaPlayback: true }}
      />
  )

**Comment by Marcoo09** [comment]
Hi @Anudnya93 
I answered you on the [issue](https://github.com/MetaLabs-inc/react-native-vimeo-iframe/issues/43#issuecomment-1484271195)
